packages/perseus/src/widgets/__tests__/interactive-graph.test.ts

```diff
@@ -154,17 +154,6 @@ describe("mafs graphs", () => {
                 renderQuestion(question, apiOptions);
             });
 
-            it("should reject when has not been interacted with", () => {
```

Comment:
Warning to reviewers: this test broke with my change and I'm worried I might be introducing a regression.

However when I run this test with Mafs turned off, the test also fails. My change makes the external Mafs API more consistent with the legacy API.

At its core, the root of the problem is that the legacy API passes back more than just the type of the initial graph; it also includes things like `numSegments`. The fact that we aren't passing that back up means we're overwriting our settings with an object that only has a `type` property.

So I _think_ my changes are correct and that this test is incorrect. I'd appreciate thoughts though.

Reply:
I think we should keep the test, because it covers important behavior. Without this test, I don't think there will be anything to ensure that we mark unanswered questions as unanswered rather than incorrect. Maybe there's a bug in `toHaveInvalidInput` that we should fix?

Reply:
So only the `point` graph is failing and it's failing for both Mafs and legacy. Console logging things shows that they exhibit the same behavior when getting user input.

I now think there was an existing issue with `point` that was getting hidden by the issue I fixed in this PR. I'll keep investigating.

Reply:
Okay @benchristel! I did a lot more research (documented in that bug ticket), here's the conclusion I same to:
- At least since before 2024 (before this work kicked off), the legacy interactive graph grades a point graph as correct if:
  - The initial state is correct
  - Even if the user hasn't interacted with the chart
- These tests are testing that circumstance: the test starts in a default state and we try to validate it before interaction
- The tests were passing because of the bug I fixed

I don't think this is the time to fix this issue because:
- It's a really unlikely edge case (why would we start the exercise with the correct answer already set?)
- Changing this part of the logic will also change the legacy behavior, possibly introducing regressions
- After we retire the legacy experience, we could refactor to include `hasBeenInteractedWith` (which is only available to Mafs right now)

If we don't want to remove these tests, I propose we change them so that this particular test is testing data that _doesn't_ begin in the correct state.

I'm going to update this PR and I've thrown time on the calendar for us to talk.